kbase-extension/static/kbase/js/widgets/narrative_core/upload/importSetup.js

```diff
@@ -7,7 +7,7 @@ define([
     'common/runtime',
     'StagingServiceClient',
     './importErrors',
-], (Promise, Jupyter, Config, APIUtil, StringUtil, Runtime, StagingServiceClient, Error) => {
+], (Promise, Jupyter, Config, APIUtil, StringUtil, Runtime, StagingServiceClient, Errors) => {
```

Comment:
`Errors` was overwriting the built-in `Error` class. Oops.

Reply:
What did built-ins ever do for anyone? Overwrite the lot of 'em, I say!